homeassistant/components/camera/synology.py

```diff
@@ -115,6 +135,9 @@ def update(self):
         self._motion_setting = self._surveillance.get_motion_setting(
             self._camera.camera_id)
         self.is_streaming = self._camera.is_enabled
+        self.home_mode = self._surveillance.get_home_mode_status()
+        self.hass.states.set('camera.synology_home_mode',
```

Comment:
This is wrong. The camera entity can only have the states defined and handled in the base camera entity class.

We could add it as a state attribute.

What does home mode mean?

Reply:
I’m not sure if home mode makes sense here. It’s a global setting and not camera specific.

@martinhjelmare It allows you to have different record settings.